### ISSUE TITLE
[ENHANCEMENT] Add database selection for Performance monitor

### ISSUE THREAD

**Issue by weafert** [CSOIreland/Server-API-Library#5]
Add database select for performance monitor

Thank you very much

**Comment by lorenzobruni** [comment]
Add the following key to the Web.config and set the value of **API_PERFORMANCE_DATABASE** to the chosen **connectionString**

```xml
<!-- Performance - Choose the Database connection string where to store the records -->
<add key="API_PERFORMANCE_DATABASE" value="defaultConnection" />
```

@weafert this will be available in the next **4.4.1** release. Cheers :-)

**Comment by lorenzobruni** [comment]
Implemented in Release 4.4.1 https://github.com/CSOIreland/Server-API-Library/releases/tag/4.4.1
